recipes/mono/mono-osx.recipe

```diff
@@ -30,6 +30,7 @@ class Recipe(recipe.Recipe):
         'lib/mono/4.0-api',
         'lib/mono/4.5',
         'lib/mono/4.5-api',
+        'lib/mono/4.7-api',
```

Comment:
add this in the devel also: https://github.com/fluendo/cerbero/pull/147/files#diff-a723132f3ff0967bba7099dd52409516R55